### PR TITLE
Marian : my first script

### PR DESCRIPTION
**Pull request by mitumarian02**

### CODE REVIEW

File: Marian/Task1_Scripting/Task1.sh [hunk 1]
```diff
@@ -0,0 +1,107 @@
+#! /bin/bash
+#FIRT TASK!!!
+#TO SWITCH BETWEEN THE IMPLEMENTATIONS USE "first" for first or "second" for second
+    #eg: Task1.sh newBranchName first 
+
+firstImplementation(){
+#*** 1-3 ***
+repoPath="/home/projectx/Desktop/gitBasics/ausy_academy_2022"
```
Comment (nbiro) [inline Marian/Task1_Scripting/Task1.sh:8]: indentation

File: Marian/Task1_Scripting/Task1.sh [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+#*** 12 ***
+for i in ./*gitMessage*
+do
+    rm $i
+done
+}
+
+secondImplementation(){
+#*** 1-3 ***
+repoPath="/home/projectx/Desktop/gitBasics/ausy_academy_2022"
```
Comment (nbiro) [inline Marian/Task1_Scripting/Task1.sh:61]: this code is the same for both implementation, please move duplicated code either to a separate function or just outside of a function

**Review by nbiro — CHANGES_REQUESTED** [review]

**Review by mitumarian02 — COMMENTED** [review]
Renamed the last commit and couldn't push without a pull that merged branches (i dont know)

**Comment by nbiro** [comment]
> Renamed the last commit and couldn't push without a pull that merged branches (i dont know)

if you rename a commit, you need to use push -f or push --force-with-lease (google the difference pls)

**Review by nbiro — APPROVED** [review]

**Pull request merged**